dialog.cpp

```diff
@@ -103,3 +133,30 @@ void Dialog::checkSpelling()
                 tr("Finished"),
                 tr("The spell check has finished."));
 }
+
+
+void Dialog::replaceAll(int nPos, QString sOld, QString sNew) {
+    QTextCursor cursor(ui->textEdit->document());
+    cursor.setPosition(nPos-sOld.length(), QTextCursor::MoveAnchor);
+
+    while(!cursor.atEnd()) {
```

Comment:
Please add a space between the while and the '('.
I know the code has some occurences where no space was used, however the official Qt coding style employs that.